docs/tasks/debug-application-cluster/debug-stateful-set.md

```diff
@@ -7,78 +7,68 @@ approvers:
 - janetkuo
 - kow3ns
 - smarterclayton
-title: Debug a StatefulSet
+title: 调试 StatefulSet
+cn-approvers:
+- chentao1596
 ---
+<!--
+title: Debug a StatefulSet
+-->
 
 {% capture overview %}
 
+<!--
 This task shows you how to debug a StatefulSet.
+-->
+本任务展示如何调试 StatefulSet。
 
 {% endcapture %}
 
 {% capture prerequisites %}
 
-
+<!--
 * You need to have a Kubernetes cluster, and the kubectl command-line tool must be configured to communicate with your cluster.
 * You should have a StatefulSet running that you want to investigate.
+-->
+* 您需要有一个 Kubernetes 集群，还要配置 kubectl 命令行工具，并且需要确保它是与您的集群进行通信。
```

Comment:
as i remember that, kubectl could run on any node or container, but we should config kubectl to make sure it could work. so configure kubectl is to communicate with kubernetes cluster.
这里感觉是 配置kubectl，用以让kubectl和cluster之间保持通讯的意思。

Reply:
Thank you for your patient analysis, ^_^!
Updated